assets/overlays/azure-disk/patches/controller_add_driver.yaml

```diff
@@ -0,0 +1,89 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  annotations:
+    config.openshift.io/inject-proxy: csi-driver
+    config.openshift.io/inject-proxy-cabundle: csi-driver
+spec:
+  template:
+    spec:
+      initContainers:
+        # Merge /etc/kubernetes/cloud.conf (on the host) with secret "azure-disk-credentials" into "merged-cloud-config" emptydir.
```

Comment:
This is for another PR & discussion, but this will not work on HyperShift at all. /etc/kubernetes/cloud.conf on the host has no relation to the Azure config + credentials that should be used to manage storage of a guest cluster.  For example, the mgmt cluster does not need to be on Azure at all.
We will need to figure out something else.

Reply:
Yeah this doesn't work on hypershift. This is one of the last blocking items. This prevents rollout of deployment in hypershift. We will have to figure something out.

Reply:
This is now fixed and tested with hypershift and it works as expected. I ended up using a certain secret in hypershift envionment.